src/pytest_postgresql/executor.py

```diff
@@ -138,6 +138,7 @@ def init_directory(self):
         self.clean_directory()
         init_directory = [self.executable, 'initdb', '--pgdata', self.datadir]
         options = ['--username=%s' % self.user]
+        env = self._envvars.copy()
```

Comment:
I'm not sure why the osx build fails now for the innitdb, but won't fail for the start command itself...

Maybe we're missing some more environment variables now. In a base class it's being done like that:
https://github.com/ClearcodeHQ/mirakuru/blob/master/src/mirakuru/base.py#L216

So we copy current envvars and update it with `self._envvars`. Alternative would be to use `self._popen_kwargs()['env']` here.

Reply:
This is not better, see https://github.com/ClearcodeHQ/pytest-postgresql/pull/334/commits/ff31a3f38bb92f7c92325db9f67941327849aa4f. The actual problem on osx is:
```
initdb: error: invalid locale settings; check LANG and LC_* environment variables
pg_ctl: database system initialization failed
```
(e.g. at https://travis-ci.org/github/ClearcodeHQ/pytest-postgresql/jobs/731374804#L414)
So it seems that `C.UTF-8` is not available on travis' osx.